faf.spec.in

```diff
@@ -143,19 +143,19 @@ Requires: %{satyr_dep}
 %description problem-ruby
 A plugin for %{name} handling ruby scripts problems.
 
-%package yum
-Summary: %{name}'s yum plugin
+%package dnf
```

Comment:
Not sure, but won't this become a hell when updating? There will be package faf-yum, that requires faf in the same version as is the package, but we want to install a new version. Maybe "obsoletes faf-yum"?

Reply:
Thank you for the review and comments. I have fixed the issue, it updated in my next push.